enterprise/tpa_pipeline.py

```diff
@@ -61,6 +61,7 @@ def handle_enterprise_logistration(backend, user, **kwargs):
         **kwargs: Any remaining pipeline variables
 
     """
+    LOGGER.info(f'Beginning enterprise logistration for user {user.id}')
```

Comment:
in this log, it may be better to also replicate wording on like `LMS User Id` similar to the other one

Reply:
yeah I can add that. I specifically added the word LMS there though because the log is on ECU which also has an id field and is of the user variety.